checkstyle-tester/diff.groovy

```diff
@@ -289,6 +291,27 @@ def generateSummaryIndexHtml(diffDir, checkstyleBaseReportInfo, checkstylePatchR
     println 'Creating report summary page finished...'
 }
 
+def generateListOfProjectsHtml(diffDir, listOfProjects) {
+   def diffToolDir = Paths.get("").toAbsolutePath()
+        .parent
+        .resolve("patch-diff-report-tool")
+        .toFile()
+    def diffToolJarPath = getPathToDiffToolJar(diffToolDir)
+    this.class.classLoader.rootLoader.addURL(new URL("file:$diffToolJarPath"))
+    def textTransform = Class.forName("com.github.checkstyle.site.TextTransform").newInstance()
```

Comment:
Please add a line separation between loading the JAR and executing the class. It makes things easier to read when you have a separation between thoughts.